docs/diagrams/src/readme.md

```diff
@@ -0,0 +1,41 @@
+# Diagram Sources #
+
+Store your source files in this directory so that, as the architecture evolves,
+the diagrams can change and updated renderings can replace previous versions 
+in ../src and ../out.
+
+## Guidelines ##
+
+* Store your source code (as .wsd files) in the /docs/diagrams/src directory.
```

Comment:
Did you mean store your UML diagrams?

Reply:
it's in the docs readme, so i figured it was implied, but it's good to be explicit